### ISSUE TITLE
v2.0 has some issues

### ISSUE THREAD

**Issue by nikivdev** [deanishe/alfred-stackoverflow#7]
Firstly the release has `.alfred4workflow` as extension ending and thus is not openable by Alfred. Had to change it to `.alfredworkflow` to be recognizable by Alfred and be opened.

Then when I did load it, trying to run the `.so` script filter I get error in console:

```
[17:23:49.332] ERROR: StackExchange Search[Script Filter] Code 1: Traceback (most recent call last):
  File "so.py", line 27, in <module>
    from common import CLIENT_ID, CLIENT_KEY
ImportError: No module named common
```

But not CLIENT_ID and KEY are setup to be defined in wf env variables. Perhaps Im missing something.

Get same error on `stack` keyword too.

Just noticed it also says `ImportError: No module named common` so perhaps I have to pip install it inside workflow, not sure.

**Comment by deanishe** [comment]
Thanks for finding the issues. The import problems are fixed in [v2.0.1](https://github.com/deanishe/alfred-stackoverflow/releases/tag/v2.0.1).

The `.alfred4workflow` issue is an Alfred bug. Andrew will have to fix that.

**Comment by deanishe** [comment]
> Firstly the release has `.alfred4workflow` as extension ending and thus is not openable by Alfred

Should be supported in the next version of Alfred.
